src/ToolBox/superpmi/superpmi-shared/lightweightmap.h

```diff
@@ -329,9 +329,9 @@ class LightWeightMap : public LightWeightMapBuffer
                 }
         */
         // O(log n) version
-        int first = 0;
-        int mid   = 0;
-        int last  = numItems - 1;
+        unsigned int first = 0;
+        unsigned int mid   = 0;
+        unsigned int last  = numItems - 1;
```

Comment:
This change breaks the case when numItems is 0. I assume the compiler was complaining on this line, so I'd just add cast here.

Reply:
done